src/components/FileList/FileList.js

```diff
@@ -281,6 +293,25 @@ export default class FileList extends PureComponent {
     )
   }
 
+  renderLicenseCell(record) {
```

Comment:
Same binding nit

Reply:
changed to use arrow function instead. See https://github.com/clearlydefined/website/pull/980/commits/ea7609b1515cce98401db78882df1a4c788e3872